apps/builder/app/builder/features/workspace/canvas-tools/outline/collaborative-instance-outline.tsx

```diff
@@ -0,0 +1,20 @@
+import { useStore } from "@nanostores/react";
+import {
+  $collaborativeInstanceOutlineAndInstance,
+  $collaborativeInstanceSelector,
+} from "~/shared/nano-states";
+import { Outline } from "./outline";
+import { applyScale } from "./apply-scale";
+import { scaleStore } from "~/builder/shared/nano-states";
+
+// Outline of an instance that is being edited by AI or a human collaborator.
+export const CollaborativeInstanceOutline = () => {
+  const selectedInstanceSelector = useStore($collaborativeInstanceSelector);
```

Comment:
`selectedInstanceSelector` it's not selectedInstanceSelector

Reply:
And seems like no reason to use it here,
$collaborativeInstanceOutlineAndInstance is enough